src/bionics.cpp

```diff
@@ -389,8 +389,9 @@ bool player::activate_bionic( int b, bool eff_only )
         bool extracted = false;
         for( auto it = g->m.i_at(pos()).begin();
              it != g->m.i_at(pos()).end(); ++it) {
+            static const auto volume_per_water_charge = units::from_milliliter( 500 );
```

Comment:
Previous logic is `it->volume() / 2`, now it must be `it->volume() / 500` because 2 in current system represents 500 ml. I have no idea why this exact value had been chosen.